web-app/src/pages/Login.vue

```diff
@@ -47,44 +47,22 @@
 </template>
 
 <script lang="ts">
-import api from 'src/services/http';
+import api from './../services/API';
 import { Vue, Component } from 'vue-property-decorator';
-import axios, { AxiosResponse } from 'axios';
 
 @Component
 export default class Login extends Vue {
-  private logoUrl: unknown = '';
+  private logoUrl = '';
 
   private async loginWithMS() {
-    const authUrlResponse: AxiosResponse = await axios({
-      method: 'POST',
-      url: 'http://localhost:3000/v1/users/login',
-    });
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-    window.location = authUrlResponse.data.data as Location;
+    const authUrlResponse: Location = await api.auth.getLoginUrl();
+    window.location = authUrlResponse;
```

Comment:
```suggestion
    const authUrl= await api.auth.getLoginUrl();
    window.location = authUrl;
```

Type location is redundant bcz return type is this. In case of simple response (not complex/raw object) we can skip `Response` postfix.
Hmm may rename function to `redirectToMSAuth` or something like that